.travis.yml

```diff
@@ -1,18 +1,37 @@
 language: python
+
 python:
   - "2.6"
   - "2.7"
   - "3.2"
   - "3.3"
   - "3.4"
+  - "3.5"
   - "pypy"
+  - "pypy3"
+
+# Use container-based infrastructure
+sudo: false
+
 install:
   - if [[ $TRAVIS_PYTHON_VERSION == '2.6' ]]; then pip install unittest2; fi
   - pip install .
   - pip install -r requirements.txt
+
+  # Coverage 4.0 doesn't support Python 3.2
+  - if [ "$TRAVIS_PYTHON_VERSION" == "3.2" ]; then travis_retry pip install coverage==3.7.1; fi
+  - if [ "$TRAVIS_PYTHON_VERSION" != "3.2" ]; then travis_retry pip install coverage; fi
+
 script:
-  - if [[ $TRAVIS_PYTHON_VERSION == '2.6' ]]; then unit2 discover; fi
-  - if [[ $TRAVIS_PYTHON_VERSION != '2.6' ]]; then python -m unittest discover; fi
-branches:
-  only:
-    - master
+  - if [[ $TRAVIS_PYTHON_VERSION == '2.6' ]]; then coverage run --source=langdetect --omit=langdetect/tests/* -m unittest2 discover; fi
+  - if [[ $TRAVIS_PYTHON_VERSION != '2.6' ]]; then coverage run --source=langdetect --omit=langdetect/tests/* -m unittest discover; fi
+
+after_success:
+ - pip install coveralls
+ - coveralls
+
+after_script:
+ - coverage report
+ - pip install pep8 pyflakes
```

Comment:
I think it's more valuable to see the output of `pyflakes` (possible programming errors) and `pep8` (possible style problems) separately. 

And installing `flake8` also installs some things other than just `pyflakes` and `pep8`.